src/main/java/org/gelecekbilimde/scienceplatform/auth/config/SecurityConfiguration.java

```diff
@@ -43,6 +45,8 @@ public SecurityFilterChain filterChain(HttpSecurity httpSecurity,
 			.authorizeHttpRequests(customizer -> customizer
 				.requestMatchers(HttpMethod.GET, "/public/**").permitAll()
 				.requestMatchers("/api/v1/version", "/api/v1/auth/**", "/api/v1/category/**")
+				.permitAll()
+					.requestMatchers(HttpMethod.GET, "/api/v1/users/*/followers", "/api/v1/users/*/followings")
```

Comment:
Authenticated olmayan kullanıcılar takip edememeli, takip yapabilmesi için sisteme giriş yapması gerekiyor bu yüzden access düzenlemesi permitAll'dan ziyade controllerda halledilmeli.

Reply:
Takip etmiyorlar zaten. Onlar takipçi/takip edilen görüntüleme endpointleri